taskana-adapter-camunda-tasklistener/src/main/java/pro/taskana/adapter/camunda/tasklistener/TaskanaTaskListener.java

```diff
@@ -0,0 +1,114 @@
+package pro.taskana.adapter.camunda.tasklistener;
+
+import org.camunda.bpm.engine.delegate.DelegateTask;
+import org.camunda.bpm.engine.delegate.TaskListener;
+import org.camunda.bpm.engine.impl.context.Context;
+import org.camunda.bpm.model.bpmn.instance.camunda.CamundaProperties;
+import org.camunda.bpm.model.bpmn.instance.camunda.CamundaProperty;
+
+import java.sql.*;
+import java.time.Instant;
+import java.util.Collection;
+import java.util.logging.Logger;
+
+public class TaskanaTaskListener implements TaskListener {
+
+    private final Logger LOGGER = Logger.getLogger(this.getClass().getName());
+
+    private static TaskanaTaskListener instance = null;
+
+    public static TaskanaTaskListener getInstance() {
+        if (instance == null) {
+            instance = new TaskanaTaskListener();
+        }
+        return instance;
+    }
+
+    public void notify(DelegateTask delegateTask) {
+
```

Comment:
I would recommend to use also in this case the try-with-resources syntax. This removes the need for the inner try-catch block. In addition, all Exceptions (not only SQLExceptions) should be caught...